tests/cross/src/test/scala/tests/pc/AutoImportsSuite.scala

```diff
@@ -263,22 +298,10 @@ class AutoImportsSuite extends BaseAutoImportsSuite {
       """val p: <<Path>> = ???
         |""".stripMargin
     ),
-    // Import added *before* the wrapper here…
-    // This *seems* wrong, but the logic converting the scala file
-    // edits to sc file edits will simply add it at the beginning of
-    // the sc file, as expected.
```

Comment:
There were a need to distinguish amm sc files from scala-cli so in addition I fixed first import for ammonite.